docs/book/src/SUMMARY.md

```diff
@@ -79,6 +79,7 @@
 
   - [Generating CRDs](./reference/generating-crd.md)
   - [Using Finalizers](./reference/using-finalizers.md)
+  - [Watching Objects](./reference/watching-objects.md)
```

Comment:
```suggestion
  - [Observing Resources](./reference/observing-objects.md)
```

Reply:
I think we can use the word observe to make clear that we are speaking over observability and than, describe that the watch feature allows that.

Reply:
I used "Watching" because I think it's what most people would be looking for when trying to implement this logic. It's a keyword used all around Kubernetes and EtcD, as well as other distributed systems, like ZooKeeper and Consul.

Observing feels a little too passive for me, since we are keeping track of these resources in order to take actions on any changes. Observing is a little more removed, like keeping track of metrics.

Have no qualms changing "Objects" -> "Resources" though!

Reply:
+1 to the "object" -> "resource" change. I'm fine with "watching".